README.md

```diff
@@ -1,6 +1,133 @@
-# website
+# Website
+
 The website for the Civic Tech Index
 
+To develop the site, you'll need to first clone the repository on to your computer. For new Git users, see the [Using Git](#using-git) section below. <br><br>
+
+# OVERVIEW
+**Set up**
+1. [Join the Repo Team](#step-1-become-a-member-of-the-repository-team)
+
+2. [Using Git](#using-git) and [Fork the Repo](#step-2-fork-the-repository)
+
+3. [Clone to your local machine](#step-3-clone-your-online-repository-to-your-local-computer)
+
+4. [Set up Docker](#step-4-setting-up-docker)
+
+**Before you start working on an issue**
+
+5. [Read CTI's Site Architecture to get acquainted with how the website is structured](https://github.com/CTI-website-frontend/website/wiki/CTI-Site-Architecture)
+
+6. [Switch to new issue branch before you start making changes](#step-6-change-to-a-new-branch)
+
+
+**After you've worked on your issue and before you make a pull request:**
+
+7. [Check upstream before you push](#step-7-check-upstream-before-you-push).
+
+8. [No changes in the upstream repo](#step-7a-no-changes-in-the-upstream-repository)
+
+**Or**
+
+9. [Conflicting changes in the upstream repo](#step-7b-conflicting-changes-in-the-upstream-repository) and how to resolve them
+              
+**Okay. You're good to go!**        
+ 
+10. [Complete the pull request](#step-8-complete-the-pull-request)
+
+---
+
+### Forking and cloning the repository with proper security
+
+#### Step 1 Become a member of the repository Team
+
+In the `hfla-site` slack channel, send your GitHub name to the project manager (or on the slack channel thread) and we'll add you as a member to the GitHub repository Team.
+
+Once you have accepted the GitHub invite (comes via email or in your GitHub notifications), please do the following:
+
+1. Mark your own membership public https://help.github.com/en/articles/publicizing-or-hiding-organization-membership#changing-the-visibility-of-your-organization-membership
+
+1. Setup two factor authentication on your account https://github.com/hackforla/governance/issues/20
+
+
+## Using Git
+
+This section discusses some tips and best practices for working with Git.
+
+### Making changes, committing and pushing
+
+1. Generally changes start on your local clone of your fork of this repository, in your own branch.
+
+1. Commit your changes with a comment related to the issue it addresses to your local repository.
```

Comment:
Is there a way to make this section an ordered list with consecutive numbers rather than "1." for each line?

Reply:
Ah see above, this is just how it looks in markup, please check in Preview and the numbering should appear correctly.

Reply:
Ok, cool! Thanks for the explanation.